flink-contrib/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackendTest.java

```diff
@@ -122,6 +124,22 @@ protected RocksDBStateBackend getStateBackend() throws IOException {
 		return backend;
 	}
 
+	// small safety net for instance cleanups, so that no native objects are left
```

Comment:
This change actually belongs to the previous commit, so I will put it back there.